src/Stick.tsx

```diff
@@ -99,7 +99,7 @@ function Stick({
 
       const { target } = ev
       if (
-        target instanceof window.HTMLElement &&
+        target instanceof (window.HTMLElement || window.SVGElement) &&
```

Comment:
**Suggestion:** I believe we could use `window.Element`, as there's only `HTMLElement` and `SVGElement` as of today implementing `Element`.

Reply:
**Note:** Same for the type addition on L222.

Reply:
Done! I just pushed the latest change 👍🏽